README.md

```diff
@@ -112,16 +112,21 @@ For example, in dev, the CIDR is set to `10.40.0.0/16`. The Terraform manifest a
 Coming soon
 
 ## Deploy and Upgrade Ingest Applications
+Deployments are automatically handled by [Gitlab](https://gitlab.ebi.ac.uk/) but you can still manually deploy if required (see below). However, `ontology` is not deployed by Gitlab but there is a special command for deploying ontology (see below).
 
-### Deploy one kubernetes dockerized applications to an environment (aws)
+### Manually deploy one kubernetes dockerized applications to an environment (aws)
```

Comment:
All of these are manual haha

Reply:
Just emphasizing that we should only do this if we have to

Reply:
I see. You could put a note below it instead. That deployment are being done via gitlab and should avoid manual running of these commands unless necessary.

Reply:
There is a note above

Reply:
oops, has that been there before? my bad! haha okidokies!